src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+  const getSum = (a) => {
+    const isEven = countCalls % 2 === 1;
+
+    countCalls++;
+
+    return (b) => {
+      if (countCalls > 3 && isEven) {
+        return 'Bzzz... Error!';
```

Comment:
I can`t understand how`ve U passed the tests with the wrong condition... 🤔

Reply:
the same)